client/my-sites/plan-features-2023-grid/components/plan-comparison-grid.tsx

```diff
@@ -493,10 +494,18 @@ const PlanComparisonGridFeatureGroupRowCell: React.FunctionComponent< {
 	restructuredFeatures: RestructuredFeatures;
 	planName: string;
 	isStorageFeature: boolean;
-} > = ( { feature, visiblePlansProperties, restructuredFeatures, planName, isStorageFeature } ) => {
+	flowName: string;
+} > = ( {
+	feature,
+	visiblePlansProperties,
+	restructuredFeatures,
+	planName,
+	isStorageFeature,
+	flowName,
+} ) => {
 	const translate = useTranslate();
-	const highlightAdjacencyMatrix = useHighlightAdjacencyMatrix( visiblePlansProperties );
-	const highlightLabel = useHighlightLabel( planName );
+	const highlightAdjacencyMatrix = useHighlightAdjacencyMatrix( visiblePlansProperties, flowName );
+	const highlightLabel = useHighlightLabel( planName, null );
```

Comment:
Should this be `const highlightLabel = useHighlightLabel( planName, flowName );` ? 🤔